### PR TITLE
feat(ingestion): add all required ingestion endpoints

### PR DESCRIPTION
**Pull request by jtheoof**

### CODE REVIEW

File: mnubo/ingestion.go [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+type EventsExist []map[string]bool
+
+type ObjectOwnerPair struct {
+	XDeviceID  string `json:"x_device_id"`
+	Username   string `json:"username"`
+}
+
+type ClaimResult struct {
+	ID      string `json:"id"`
```
Comment (daddykotex) [inline mnubo/ingestion.go]: that's great ( a little limited but elegant)

File: mnubo/ingestion.go [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+	if err != nil {
+		return err
+	}
+
+	return e.Mnubo.doRequestWithAuthentication(cr, results)
+}
+
+func (e *Events) SendFromDevice(deviceId string, events interface{}, options SendEventsOptions, results interface{}) error {
+	cr, err := buildEventsClientRequest(events, options, fmt.Sprintf("%s/%s/events", objectsPath, deviceId))
```
Comment (daddykotex) [inline mnubo/ingestion.go]: `devicePath`? (since it's `SendFromDevice`)

Comment (daddykotex) [reply mnubo/ingestion.go]: forget it, I misread the code, here `objectsPath` is `/objects`, it's ok

**Comment by jtheoof** [comment]
Fixed the build

**Comment by jtheoof** [comment]
Yes indeed, but because the payload can vary from one namespace to another it's quite hard to type properly. So I think the SDK should be well documented (which is lacking for now) but leave the typing for unpredictable payloads to the consumer of the SDK.

**Comment by jtheoof** [comment]
It's missing one API: `UpdateOwnerPassword`. I'll update.

**Comment by jtheoof** [comment]
Added one function to update an owner password.

**Pull request merged**